ci/bazel-scripts/main.sh

```diff
@@ -6,42 +6,33 @@
 
 set -eufo pipefail
 
+# default behavior is to build targets specified in BAZEL_TARGETS and not upload to s3
```

Comment:
While you're at it. Consider changing the gitlab reference in line 3 to GitHub.

Reply:
Good point - I actually do have it covered here: https://github.com/dfinity/ic/pull/1343/files